package.json

```diff
@@ -30,6 +31,7 @@
     "node": ">=4"
   },
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
Not it all, @types/jquery is just typescript jquery definition itself. This package contains definitions (".d.ts") required for extending jquery from typescript library perspective.  

Since jquery itself and some other libraries are not typescript made. Some of the official packages does not count with d.ts within. This way, we use [@types](https://github.com/DefinitelyTyped/DefinitelyTyped) repository to fill the gaps. This is transparent for javascript. But is required for a typed javascript (typescript) fill the gap what JQuery is in typed world.  

Anyway, I think this is something that can wait if you are not really sure, and ask more people about it. I use my fork in my project, cause my project is a bootstrap+jquery+typescript+webpack web app. But i'm sure it works on non typescript too. :)